shard.lucky.yml

```
@@ -1,25 +1,22 @@
 name: openapi-generator
-version: 0.2.7
+version: 0.4.0
```

Any reason for the double bump?
```suggestion
version: 0.3.0
```

---

upstream is already 0.3.3